### DISCUSSION TITLE
Admin

### DISCUSSION THREAD

**Discussion by TooTallbmf13** [community/community#51800] [Other Feature Feedback, Questions, & Ideas]
Need help gaining Admin authority on my LG STYLO 6 from T-Mobile

**Comment by queenofcorgis** [comment]
Hi @TooTallbmf13,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on LG or T-Mobile. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
